include/geom/elem.h

```diff
@@ -322,36 +322,66 @@ class Elem : public ReferenceCountedObject<Elem>,
   bool contains_edge_of(const Elem *e) const;
 
   /**
-   * This function finds all elements (including this one) which
-   * touch the current active element at the specified point, which
-   * should be a point in the current element.
+   * This function finds all active elements (including this one)
+   * which are in the same manifold as this element and which touch
+   * the current active element at the specified point, which should
+   * be a point in the current element.
+   *
+   * Elements which are not "in the same manifold" (e.g. the
+   * interior_parent of a boundary element) will not be found with
```

Comment:
I'm not quite sure I understand this. So, let's say I have a 2D manifold (in 3D space) with a 1D manifold overlapping on the boundary of the 2D manifold. Will this then not find the 1D element overlapping?